OrbitGl/Batcher.h

```diff
@@ -63,6 +63,18 @@ struct TriangleBuffer {
   BlockChain<Color, 3 * NUM_TRIANGLES_PER_BLOCK> picking_colors_;
 };
 
+struct Buffers {
```

Comment:
nit. "Buffers" is very generic, consider renaming.

Reply:
Renamed to ShapeBuffers

Reply:
"Primitive" is usually used in this context, I would prefer "Primitive" over "Shape"

Reply:
done!